src/main/java/gov/acwi/wqp/etl/biodata/domain/BiodataOrgDataRowMapper.java

```diff
@@ -0,0 +1,24 @@
+package gov.acwi.wqp.etl.biodata.domain;
```

Comment:
This is an code organization comment. Should all of the ETLs try to follow the same package hierachy? I don't think stewards using "domain". And I know NWIS doesn't. Thoughts @dsteinich and @ssoper-usgs

Reply:
I think you're right - this package name is a carryover from an example I took from an early pass of the stewards-etl.  I'll change it to follow the stewards/nwis package name convention.